### PR TITLE
Implement Banner shop

### PR DESCRIPTION
**Pull request by Readpato**
### 🔗 Linked issue

https://github.com/Schroedinger-Hat/schroedinger-hat/issues/214

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Implement shop banner in the website. Should be merged whenever we release the LinkedIn post

image

image

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Readpato** [comment]
@TheJoin95 @404answernotfound Merge whenever you see fit :)

**Review by TheJoin95 — APPROVED** [review]

File: src/components/layout/TheBanner.vue [hunk 1]
```diff
@@ -7,20 +7,19 @@ defineProps<{
...
 const { handleDismiss, banner } = useBannerCookie()
 </script>
 
 <template>
   <div
-    v-if="false"
+    v-if="!banner.dismissed"
     class="banner"
     text="center base"
     fixed inset-x-0 bottom-0 p="2 lg:4"
```
Comment (TheJoin95) [inline src/components/layout/TheBanner.vue:18]: Are these class value correct ?

**Comment by TheJoin95** [comment]
@Readpato 
image

File: src/components/layout/TheBanner.vue (hunk 1 above)
Comment (Readpato) [reply src/components/layout/TheBanner.vue:18]: Classes are correct.
What is missing is a class that changes the color of the text whenever it's on light mode

**Comment by Readpato** [comment]
@TheJoin95  There we go, I had to do a dirty hack to display a background color for the SVG icon because the global classes declared in `global.scss` where overwriting whatever I was trying to modify.

image

image


PR is ready to be merged whenever we see fit

**Review by TheJoin95 — APPROVED** [review]
Thank!

**Pull request merged**